core/src/main/java/io/th0rgal/oraxen/mechanics/provided/gameplay/furniture/FurnitureListener.java

```diff
@@ -205,6 +205,7 @@ private FurnitureMechanic getMechanic(ItemStack item, Player player, Block place
         if (FurnitureFactory.getInstance().isNotImplementedIn(itemID) || BlockHelpers.isStandingInside(player, placed)) return null;
         if (!ProtectionLib.canBuild(player, placed.getLocation())) return null;
         if (OraxenFurniture.isFurniture(placed)) return null;
```

Comment:
Might be better to simply implement this as an additional check in isFurniture
Meant for the longest time for it to do so

Reply:
done